webapp/src/test/java/com/box/l10n/mojito/utils/BytesGZIPTest.java

```diff
@@ -12,7 +12,7 @@ public class BytesGZIPTest {
   static final byte[] DECOMPRESSED = STR.getBytes(StandardCharsets.UTF_8);
   static final byte[] COMPRESSED =
       new byte[] {
-        31, -117, 8, 0, 0, 0, 0, 0, 0, 0, 43, -50, -49, 77, 85, 40, 46, 41, -54, -52, 75, 87, 40,
+        31, -117, 8, 0, 0, 0, 0, 0, 0, -1, 43, -50, -49, 77, 85, 40, 46, 41, -54, -52, 75, 87, 40,
```

Comment:
do you know why this has changed?

Reply:
Yes, the explanation it is in the commit message
"unit test fix. GZIPOutputStream#writeHeader changed OS property from 0 to -1"